documentation/src/main/markdown/downloadingandinstalling/selfupdatingdetect.md

```diff
@@ -1,61 +1,61 @@
+## Scenarios where [detect_product_short] self update will not execute
 
-If there exists no mapping in [blackduck_product_name], or if the current version of [company_name] [solution_name] matches the mapped version in [blackduck_product_name], or any issue occurs during the execution of the Self Update feature, then [company_name] [solution_name] will continue with the current version to execute the scan.
+If there exists no mapping in [bd_product_short], or if the current version of [detect_product_short] matches the mapped version in [bd_product_short], or any issue occurs during the execution of the Self Update feature, then [detect_product_short] will continue with the current version to execute the scan.
 
-If the [company_name] [solution_name] URL of the [company_name] [solution_name] .jar file to download and run has been hardcoded via [company_name] [solution_name] property `DETECT_SOURCE` environment variable or the [company_name] [solution_name] version set by the `DETECT_LATEST_RELEASE_VERSION` or `DETECT_VERSION_KEY` variables, self update will not occur. These are optional System environment properties used by Detect upgrade scripts.
+If the [detect_product_short] URL of the [detect_product_short] .jar file to download and run has been hardcoded via [detect_product_short] property `DETECT_SOURCE` environment variable or the [detect_product_short] version set by the `DETECT_LATEST_RELEASE_VERSION` or `DETECT_VERSION_KEY` variables, self update will not occur. These are optional System environment properties used by [detect_product_short] upgrade scripts.
 
-If the [blackduck_product_name] “Internally Hosted” option has been selected and a [company_name] [solution_name] download location has not been provided, the feature will not be enabled.
+If the [bd_product_short] “Internally Hosted” option has been selected and a [detect_product_short] download location has not been provided, the feature will not be enabled.
 
-For further [blackduck_product_name] configuration information, refer to the documentation provided under the topic:
-<xref href="DetectLocation.dita" scope="peer"> Hosting location for Synopsys Detect.
+For further [bd_product_short] configuration information, refer to the documentation provided under the topic:
+<xref href="DetectLocation.dita" scope="peer"> Hosting location for [detect_product_short].
 <data name="facets" value="pubname=bd-hub"/>
 
-## [company_name] [solution_name] log examples for self update
+## [detect_product_short] log examples for self update
 
 Downgrade to prior version blocked:  
 
 ``` 
-2023-05-05 12:20:57 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.synopsys‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.   
+2023-05-05 12:20:57 EDT INFO  \[main] - Detect-Self-Updater:  Checking https://test1.blackduck‎ .com/api/tools/detect API for centrally managed Detect version to download to /Users/testuser/tmp.   
```

Comment:
Probably doesn't matter too much but there's a strange space before ` .com` that also existed before here and below in this file.

Reply:
Ah yes, that is there intentionally to invalidate the URL so that readers can't click on it.